librz/util/meson.build

```diff
@@ -112,7 +117,7 @@ else
   rz_util_sources += ['big.c']
 endif
 
-rz_util_includes = [platform_inc, include_directories('sdb/src', 'sdb')]
+rz_util_includes = [platform_inc, include_directories('sdb/src', 'sdb', '.')]
```

Comment:
Is this really necessary?

Reply:
without this ninja -C build test is failing. I guess the failure is because the path to hashtable ht_*.h is not specified. So I added '.'